generators/server/templates/bundle/plugins/myplugin.yaml.ejs

```diff
@@ -1,3 +1,4 @@
+descriptorVersion: v3
```

Comment:
Do we need to do the same with the bundle descriptor and the widget descriptors?

Reply:
@firegloves do I have to add the version also in the widget descriptors?

Reply:
@avdev4j @alepintus no, this update is limited to the plugin descriptor

Reply:
no, only the plugin descriptor is interested by this change

Reply:
thanks :D